app/components/Metric.jsx

```diff
@@ -15,16 +15,20 @@ class Metric extends MetricTemplate {
           largest: Math.max(dataSize.largest, data.size),
     }
     */
-
+    // Labels used in the Metric component
     const labels = ['Average', 'Smallest', 'Largest'];
 
+    // Calculate data to be displayed in each Label
     const processedData = [
       this.calculateAvg(sum, numOfDataPoints),
       smallest,
       largest,
     ];
 
+    // Generates the chart data for the chart
```

Comment:
maybe specify that this method and the one for chartOptions are coming from the parent MetricTemplate class